content/get-started/index.md

```diff
@@ -97,7 +97,7 @@ We also need to change `index.html` to expect a single bundled js file.
   </head>
   <body>
 -   <script src="app/index.js"></script>
-+   <script src="dist/bundle.js"></script>
++   <script src="../dist/bundle.js"></script>
```

Comment:
Why is this change necessary? The file structure at this point is:
* webpack-demo/index.html
* webpack-demo/app/index.js
* webpack-demo/dist/bundle.js